modules/navier_stokes/src/fvkernels/INSFVMixingLengthReynoldsStress.C

```diff
@@ -38,15 +38,13 @@ INSFVMixingLengthReynoldsStress::INSFVMixingLengthReynoldsStress(const InputPara
     _dim(_subproblem.mesh().dimension()),
     _axis_index(getParam<MooseEnum>("momentum_component")),
     _u_var(dynamic_cast<const INSFVVelocityVariable *>(
-        &_subproblem.getVariable(_tid, params.get<std::vector<VariableName>>("u").front()))),
-    _v_var(params.isParamValid("v")
-               ? dynamic_cast<const INSFVVelocityVariable *>(&_subproblem.getVariable(
-                     _tid, params.get<std::vector<VariableName>>("v").front()))
-               : nullptr),
-    _w_var(params.isParamValid("w")
-               ? dynamic_cast<const INSFVVelocityVariable *>(&_subproblem.getVariable(
-                     _tid, params.get<std::vector<VariableName>>("w").front()))
-               : nullptr),
+        getFieldVar(params.get<std::vector<VariableName>>("u").front(), 0))),
+    _v_var(isParamValid("v") ? dynamic_cast<const INSFVVelocityVariable *>(getFieldVar(
+                                   params.get<std::vector<VariableName>>("v").front(), 0))
```

Comment:
you can just pass "v" to getFieldVar, see INSFVMomentumAdvection

Reply:
I see now. Thanks